### PR TITLE
homework02 in Myclass03

### PR DESCRIPTION
**Pull request by lishuang0821**
edit exercise 05 and 06 in class02 folder

### CODE REVIEW

**Comment by lishuang0821** [comment]
added a new pull request for homework2

**Comment by lishuang0821** [comment]
it contained exercise5, 6, and 7

File: class03/exercise_7.py [hunk 1]
```diff
@@ -31,9 +32,20 @@ def check_path(path):
 	# if doesn't exist
 	>>> False, []
 	'''
-
-
-def read_csv(file):
+	import os
```
Comment (emma-oc) [inline class03/exercise_7.py:35]: common practice is to put all imports at the very begeining of the file (e.g. line 6 )

File: class03/exercise_7.py [hunk 2]
```diff
@@ -93,8 +117,17 @@ def read_json(file):
...
+	return js
 
 
-if __name__=="__main__":
+if __name__ == "__main__":
+	check_path('AMZN.csv')
+	print(read_csv('AMZN.csv'))
+	write_csv(data_list=[(1, 2, 3, 4), (5, 6, 7, 8), (9, 10, 11, 12)], output_file='example.csv')
+	print(read_json('some.json'))
 	pass
```
Comment (emma-oc) [inline class03/exercise_7.py:133]: since you already have code here, you don't need the `pass` as placeholder here

**Comment by emma-oc** [comment]
Good job! some minor fixes

**Pull request closed without merging**